event/event.go

```diff
@@ -24,6 +25,21 @@ type Body[T any] struct {
 	Event   T      `json:"event"`
 }
 
+// Message represents a raw message received on a subscription.
+type Message struct {
+	body []byte
```

Comment:
This can also be used by the outside, should be public, shouldn't it?

Reply:
It is accessed through a method to guarantee it is read only/immutable.

Reply:
I usually declare all types and then start declaring the methods..but it is only an old habit.